docs/source/api/v2/riak_bucket_bucket_key_key_values.rst

```diff
@@ -0,0 +1,84 @@
+..
+..
+.. Licensed under the Apache License, Version 2.0 (the "License");
+.. you may not use this file except in compliance with the License.
+.. You may obtain a copy of the License at
+..
+..     http://www.apache.org/licenses/LICENSE-2.0
+..
+.. Unless required by applicable law or agreed to in writing, software
+.. distributed under the License is distributed on an "AS IS" BASIS,
+.. WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+.. See the License for the specific language governing permissions and
+.. limitations under the License.
+..
+
+.. _to-api-riak-bucket-bucket-key-key-values:
+
+*********************************************
+``riak/bucket/{{bucket}}/key/{{key}}/values``
+*********************************************
+Retrieves the `object <https://docs.riak.com/riak/kv/latest/learn/concepts/keys-and-objects/index.html#objects>`_ stored under a given `key <https://docs.riak.com/riak/kv/latest/learn/concepts/keys-and-objects/index.html#keys>`_ from a given `bucket <https://docs.riak.com/riak/kv/latest/learn/concepts/buckets/index.html>`_ in :ref:`tv-overview`.
+
+``GET``
+=======
+:Auth. Required: Yes
+:Roles Required: "admin"
+:Response Type:  Object
+
+Request Structure
+-----------------
+.. table:: Request Path Parameters
+
+	+--------+------------------------------------------+
+	| Name   | Description                              |
+	+========+==========================================+
+	| bucket | The bucket that the key is stored under  |
+	+--------+------------------------------------------+
+	| key    | The key that the object is stored under  |
+	+--------+------------------------------------------+
+
+.. code-block:: http
+	:caption: Request Example
+
+	GET /api/2.0/riak/bucket/ssl/key/demo1-latest/values HTTP/1.1
+	User-Agent: python-requests/2.22.0
+	Accept-Encoding: gzip, deflate
+	Accept: */*
+	Connection: keep-alive
+	Cookie: mojolicious=...
+
+Response Structure
+------------------
+The response structure varies according to what is stored.
```

Comment:
are there any restrictions, or is the set of response structures unbounded?

Reply:
Specified restrictions in 0d68da42bd.